paths/git-trouble/fc-scenarios.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Scenarios
+permalink: /git-trouble/01
+facilitator: false
+sidebar:
+  nav: "advanced"
+main-content: |  
+  This course is comprised of a handful of scenarios that you might find yourself in while working with Git. Before getting started with the scenarios though, you should perform the following:
+
+    - Create a test repository on [GitHub.com](https://www.github.com)
+    - Clone the test repository to your local machine
```

Comment:
Thoughts on "fresh" or "blank" instead of "test"? 

My thought is that we would avoid confusion from users who might think "test" is a type of repository.